### PR TITLE
Add tweets

### PR DESCRIPTION
**Pull request by fabryscript**

### CODE REVIEW

File: components/sections/stats-with-tweets/index.tsx [hunk 1]
```diff
@@ -36,6 +38,9 @@ const stats: StatCard[] = [
 ];
 
 export default function StatsWithTweets() {
+  const upperHalf = tweets.slice(0, 13);
```
Comment (DavideSegullo) [inline components/sections/stats-with-tweets/index.tsx:41]: Can we move this two lines outside of the component? They're just static data

File: components/sections/stats-with-tweets/index.tsx [hunk 2]
```diff
@@ -119,13 +112,20 @@ const StatCard = ({ bottleUri, iconUri, title, value, bgClass }: StatCard) => {
...
+  username,
+  userhandle,
+  date,
+  meatDetails,
+  tweetLink,
+}: { className?: string } & Partial<(typeof tweets)[0]>) => {
   return (
     <Link
-      href={"#"}
+      href={tweetLink ?? "#"}
```
Comment (DavideSegullo) [inline components/sections/stats-with-tweets/index.tsx:125]: Can we also add a `rel` prop with `external` value?

[reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel#external)

File: components/sections/stats-with-tweets/index.tsx [hunk 3]
```diff
@@ -138,11 +138,11 @@ const Tweet = ({ className }: { className?: string }) => {
...
           <div className="flex flex-col gap-1">
-            <span className="font-semibold leading-none text-neutral-100">
-              User
+            <span className="font-semibold leading-4 text-neutral-100">
+              {username}
             </span>
-            <span className="text-sm font-light leading-none text-neutral-100/60">
-              @handler · Nov 27, 2021
+            <span className="text-sm font-light text-neutral-100/60">
+              {userhandle} · {date}
```
Comment (DavideSegullo) [inline components/sections/stats-with-tweets/index.tsx:145]: We need to use the right data format like `Nov 27, 2021`. You can use [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat), you don't need to modify the json file value but you only need to format it.

File: components/sections/stats-with-tweets/index.tsx [hunk 4]
```diff
@@ -62,28 +67,16 @@ export default function StatsWithTweets() {
...
-                >
-                  {Array(8)
-                    .fill(null)
-                    .map((_, i) => (
-                      <Tweet key={`tweet ${i}`} />
-                    ))}
-                </div>
+            <div className="tweets-row-marquee-animation tweets-upper-half relative flex gap-2 transition-transform lg:gap-4">
+              {upperHalf.concat(upperHalf).map((tweet, i) => (
+                <Tweet key={`tweet ${i}`} {...tweet} />
```
Comment (DavideSegullo) [inline components/sections/stats-with-tweets/index.tsx:72]: We can use `tweetLink` as unique key

File: lib/tweets.json [hunk 5]
```diff
@@ -0,0 +1,245 @@
+[
```
Comment (DavideSegullo) [inline lib/tweets.json:1]: If we want we can just use a js array instead of reading a json file, like we do for other static contents

**Review by DavideSegullo — CHANGES_REQUESTED** [review]

File: components/sections/stats-with-tweets/index.tsx (hunk 3 above)
Comment (fabryscript) [reply components/sections/stats-with-tweets/index.tsx:145]: We also have date-fns' `format` available, wouldn't it be better?

Comment (fabryscript) [reply components/sections/stats-with-tweets/index.tsx:145]:
```tsx
{userhandle} · {format(date!, "MMM dd, yyy")}
```

**Review by DavideSegullo — APPROVED** [review]

**Pull request merged**